public/style.css

```diff
@@ -96,19 +106,32 @@ body {
   -webkit-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+
+.sidebar-btn {
```

Comment:
Can you also add `font-size: 30px` here? That will make it a bit bigger.

Reply:
Will do this in next commit.